pyproject.toml

```diff
@@ -13,7 +13,6 @@ skips = ["B105"]
 python = "^3.11"
 cssmin = "0.2.0"
 flask-compress = "1.14"
-flask-limiter = "3.5.1"
```

Comment:
can you also remove the reference to it in the readme

Reply:
removed references of 'RATELIMIT_HEADERS_ENABLED' and 'RATELIMIT_STORAGE_URI' variables from README.md, base_config.py, test_config.py and env.env_var.template files